Dockerfile

```diff
@@ -16,11 +16,15 @@ RUN wget https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 && \
 RUN npm install -g fakeit && \
     rm -rf /tmp/* /var/tmp/*
 	
+# Install nodejs couchbase package
+RUN npm install -g couchbase
```

Comment:
started getting this error `Cannot find module 'couchbase' require` at this call in the nodejs script `require('couchbase')`	so installing it through npm